boot/zephyr/Kconfig

```diff
@@ -135,7 +135,15 @@ config BOOT_BOOTSTRAP
 	  If unsure, leave at the default value.
 
 config BOOT_ENCRYPT_RSA
-	bool "Support for encrypted upgrade images"
+	bool "Support for encrypted upgrade images using RSA"
+	default n
+	help
+	  If y, images in the secondary slot can be encrypted and are decrypted
```

Comment:
Random: might be better to go into a bit more detail about the RSA vs ECIES algorithms used in the help here instead of copy/pasting the same text.

Reply:
The copy/paste remains but I also added a bit of explanation of why on option is different from the other.

Reply:
Looks great, that's a clear distinction with pointers to more info. Thanks!